eval.py

```diff
@@ -0,0 +1,117 @@
+import torch
+import numpy
+from omnigan.utils import load_opts
+from pathlib import Path
+from argparse import ArgumentParser
+from omnigan.trainer import Trainer
+from omnigan.data import pil_image_loader
+from torchvision import transforms as trsfs
+import torchvision.utils as vutils
+import torchvision.transforms.functional as TF
+import os
+
+
+def parsed_args():
+    """Parse and returns command-line args
+
+    Returns:
+        argparse.Namespace: the parsed arguments
+    """
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--config",
+        default="./shared/trainer/defaults.yaml",
+        type=str,
+        help="What configuration file to use to overwrite default",
+    )
+    parser.add_argument(
+        "--default_config",
+        default="./shared/trainer/defaults.yaml",
+        type=str,
+        help="What default file to use",
+    )
+    parser.add_argument(
+        "--path_to_images",
+        type=str,
+        help="Path of images to be inferred",
+        required=True,
+    )
+    parser.add_argument(
+        "--checkpoint",
+        type=str,
+        help="Path to experiment folder containing checkpoints/latest_ckpt.pth",
+        required=True,
+    )
+    parser.add_argument(
+        "--new_size", type=int, help="Size of generated masks",
+    )
+    parser.add_argument(
+        "--output_dir",
+        default="./output_masks/",
+        type=str,
+        help="Directory to write images to",
+    )
+
+    return parser.parse_args()
+
+
+if __name__ == "__main__":
+    # -----------------------------
+    # -----  Parse arguments  -----
+    # -----------------------------
+
+    args = parsed_args()
+    output_dir = Path(args.output_dir)
+    output_dir.mkdir(exist_ok=True)
```

Comment:
you could add `parents=True` so that we could specify a hierarchy without checking all parents exist

Reply:
True